setup.py

```diff
@@ -40,16 +40,16 @@ def _install_requirements():
     return requirements
 
 setup(
-    name='atomicapp',
-    version='0.1.9',
```

Comment:
could you read the value from https://github.com/projectatomic/atomicapp/blob/master/atomicapp/constants.py#L26 ?